cmd/ocitools/validate.go

```diff
@@ -98,531 +54,15 @@ var bundleValidateCommand = cli.Command{
 			return fmt.Errorf("The root path %q is not a directory.", rootfsPath)
```

Comment:
I think we want to move the UTF-8 checking and such into the library as well.  We'd have `NewValidator` ([or `NewConfigValidator`][1], or whatever):

    func NewValidator(spec rspec.Spec, bundlePath string, hostSpecific bool) Validator

and also:

    func NewValidatorFromPath(bundlePath string, hostSpecific bool) Validator

which did the read-in, UTF-8 check, etc., and then called `NewValidator`.

[1]: https://github.com/opencontainers/ocitools/pull/188/files#r77942970